### ISSUE TITLE
Choose Language in landing page

### ISSUE THREAD

**Issue by luca-bertero** [Nrosa01/pmd-quiz-online#9]
I think it would be nice to also let the user choose the language they want with the autodetect mode. 
For example with tiny flag on the landing page for choosing it

**Comment by Nrosa01** [comment]
Yes, I agree with that, in fact that feature was already planned along other ones. I will also use local storage to keep user prefs between sessions. But before adding new stuff I need to improve and clean the current code. After that I'll add this ^^

**Comment by luca-bertero** [comment]
Great! Can't wait to see the result!

**Comment by Nrosa01** [comment]
This will be added soon, I'm sorry it took me so long, it was easy to add but I just couldn't find the time or will to do it (also because I wanted to add other stuff). If everything goes right I'll have this by the end of this week
